cv/CV-guide.md

```diff
@@ -0,0 +1,171 @@
+```
+{
+    "accession": "QC:4000113",
+    "name": "MZ acquisition range",
+    "value": {
+	    "Sample rates": [5,3],
+	    "Counts": [2,3],
+	    "Peptide sequences": ["DYHTVLGAR AGLVTHYDR","IQACSGNDLQQWAR AWQQLDNGSCAQIR MATSGMMTELGK"],
+    },
+}
+```
+
+
+# Define a new metric term for a
+If a desired metric is not yet defined, you will need a cv term definition before you can write a metric object. 
+
+## single value, n-tuple, matrix:
+This is straight forward for these types. 
+1. Make sure, the term is not defined yet in the QC ontology
+2. Define a reasonable descriptive name while keeping concise
+3. Describe the term content, context (relations), and usage 
+4. Define its value type 
+
+Say we want to define a metric for the fastest frequency for MS/MS acquisition. We look up the QC ontology and for the purpose of this document pretend, no such term exists. We coin the name `Maximal MS2 frequency` and define it as `"The fastest frequency for MS/MS collection in any minute over the complete run"`.  This will clearly be a single value type, we do not need any spectrum identifications to assign its value, but it will be for the MS2 tandem spectra of one particular run.  And it will clearly be of type frequency, in particular the sampling frequency of spectra per minute. Hence the might look like this.
+
+```
+[Term]
+id: QC:4000061
+name: Maximal MS2 frequency
+def: "The fastest frequency for MS/MS collection in any minute over the complete run" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000024 ! MS2 metric
+relationship: has_relation MS:1000029 ! sampling frequency
+relationship: has_relation MS:1000580 ! MSn spectrum
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+```
+where the content, context (relations), and usage are described by the is_a and relationship expressions. The unit is implied as `has_units UO:0000106 ! hertz` per the `has_relation MS:1000029 ! sampling frequency` relationship.
+
+***Issue***: do we want to allow these indirect unit associations? We can be flexible, and allow unit definition individually per object instance, or mandate. But indirect is somewhere inbetween.
+
+## tables
+In case multiple types are involved,  the procedure changes slightly. 
+
+`Defined value type`s are no renamed unit types! In case you need multiple columns of the same type, it would not be helpful to have a `Defined value type` to be the same as a unit as this would result in column name conflicts. For this purpose, `Defined value type`s are 'named' stand-ins for unit types where same type occurrences are expected frequently. This also makes automated parsing a lot easier, as now each column has a dedicated name, and well defined/describet content, too, thanks to the defintion of the `Defined value type`.
```

Comment:
`Defined value type` comes a bit out of nowhere.
Mention that its a CV term: `QC:4000082`.
But otherwise I'm not sure what this paragraph wants to say.

How about: `Each column in your table requires a separate CV term, which is derived from 'Defined value type' (QC:4000082), e.g. one column could be 'Sampling rate' (QC:4000089), another one a 'Count' (...).
This ensures that every column has a unique name (thus order is not important) and well defined content.`